File: class20/cpp/AP2001-PadraoAsteriscos.cpp
#include <iostream>
int main() {
    const int N = 5;
    // Triangulo crescente (loops aninhados)
    for (int i = 1; i <= N; i++) {
        for (int j = 1; j <= i; j++) std::cout << "*";
        std::cout << "\n";
    }
    std::cout << "\n";
    // Triangulo decrescente
    for (int i = N; i >= 1; i--) {
        for (int j = 1; j <= i; j++) std::cout << "*";
        std::cout << "\n";
    }
    return 0;
}

File: class20/java/AP2001-PadraoAsteriscos.java
public class PadraoAsteriscos {
    public static void main(String[] args) {
        final int N = 5;
        // Triangulo crescente (loops aninhados)
        for (int i = 1; i <= N; i++) {
            for (int j = 1; j <= i; j++) System.out.print("*");
            System.out.println();
        }
        System.out.println();
        // Triangulo decrescente
        for (int i = N; i >= 1; i--) {
            for (int j = 1; j <= i; j++) System.out.print("*");
            System.out.println();
        }
    }
}
